src/algebra/opposites.lean

```diff
@@ -364,3 +364,9 @@ rfl
 lemma units.coe_op_equiv_symm {R} [monoid R] (u : (units R)ᵒᵖ) :
   (units.op_equiv.symm u : Rᵒᵖ) = op (u.unop : R) :=
 rfl
+
+@[simps]
+def monoid_hom.op {α β} [monoid α] [monoid β] (f : α →* β) : αᵒᵖ →* βᵒᵖ :=
```

Comment:
The linter will want this to have a docstring - it should explain how this function is different to `monoid_hom.to_opposite`